R/tidy.R

```diff
@@ -49,6 +49,7 @@ tidy.frosting <- function(x, number = NA, id = NA, ...) {
   # add id = NA as default. If both ID & number are non-NA, error.
   # If number is NA and ID is not, select the layer with the corresponding
   # ID. Only a single ID is allowed, as this follows the convention for number
+  rlang::check_dots_empty()
```

Comment:
usually [not checked](https://github.com/tidymodels/recipes/blob/36a1307656ea7804eff127fd2294cc1f2dd9421e/R/impute_linear.R#L245)